rosetta-testnet/start_stack.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -e
+
+#Color to the people
+CYAN='\x1B[0;36m'
+GREEN='\x1B[0;32m'
+NC='\x1B[0m'
+
+declare -a NODES=("0" "1" "2" "metachain")
+
+STACK_FOLDER=~/MyObservingSquadTestnet
+KEYS_FOLDER=${STACK_FOLDER}/keys
+
+#Create the folder structure for the observer stack
+mkdir -p ${STACK_FOLDER}/{proxy,node-0,node-1,node-2,node-metachain}/{config,logs}
+mkdir -p ${STACK_FOLDER}/{node-0,node-1,node-2,node-metachain}/db
+mkdir -p ${KEYS_FOLDER}
+
+#Generate Keys and place them in their respective folders
```

Comment:
do we generate new keys every time?

Reply:
I'm not sure. Maybe @andreibancioiu has the answer ?